### ISSUE TITLE
[Feature request] Capture segment

### ISSUE THREAD

**Issue by yannickhs123** [chhoumann/PodNotes#40]
I want to thank you for your awesome work.

Request: A function/hotkey to capture a segment of the podcast (ex. hotkey for timestamp +"1" to capture last 10s, +"2" to capture segment of the last 20s etc. A feature I really liked in the Airr app and would love to see it available in Obsidian !

**Comment by keerah** [comment]
Yes would be cool to have this feature to avoid keeping entire episodes!
I'd suggest using ffmpeg for segmentation perhaps.
